daphne_worker/src/durable/helper_state_store.rs

```diff
@@ -103,26 +71,24 @@ impl HelperStateStore {
             ))),
         }
     }
-}
 
-impl DapDurableObject for HelperStateStore {
-    type DurableMethod = bindings::HelperState;
+    fn should_cleanup_at(&self) -> Option<ScheduledTime> {
+        const VAR_NAME: &str = "DAP_DURABLE_HELPER_STATE_STORE_GC_AFTER_SECS";
+        static SELF_DELETE_AFTER: OnceLock<Duration> = OnceLock::new();
 
-    #[inline(always)]
-    fn state(&self) -> &State {
-        &self.state
-    }
-
-    #[inline(always)]
-    fn deployment(&self) -> DaphneWorkerDeployment {
-        self.config.deployment
-    }
-}
+        let duration = SELF_DELETE_AFTER.get_or_init(|| {
+            Duration::from_secs(
+                self.env
+                    .var(VAR_NAME)
+                    .unwrap_or_else(|e| panic!("{VAR_NAME} not defined: {e}"))
```

Comment:
Should we have a default value here like I did for `AggregateStore`? I kept the old behavior but it might not be desirable.

Reply:
Setting a default sounds good to me.